vpr/src/device/rr_graph/rr_graph_obj_utils.h

```diff
@@ -0,0 +1,161 @@
+#ifndef RR_GRAPH_OBJ_UTILS_H
```

Comment:
I did compare to netlist_utils.h. They are different in terms of the template.
The netlist_utils.h uses vtr::vector while the rr_graph_obj_utils.h uses Container.
I create this file to avoid strong conflicts with the netlist_utils.h.
We can think about more generic utils for the StrongId class, covering both netlist_utils.h and rr_graph_obj_utils.h. We can just simply merge the two files in a strong_id_utils.h. 
But where should we place the source/head files?
Looking forward to your suggestion. I can do another pull request to resolve this issue.

Reply:
Its probably best to delay this change (since it effects non-RR graph stuff) as a follow up after this PR is merged.

However here are my thoughts:

> The netlist_utils.h uses vtr::vector while the rr_graph_obj_utils.h uses Container.

I think we should then be able to just use the rr_graph_obj_utils.h versions, since if they just expect a container then they should work with vtr::vector as well.

> We can just simply merge the two files in a strong_id_utils.h.
> But where should we place the source/head files?

I'd probably put them under vpr/src/util.